app/views.py

```diff
@@ -51,8 +64,23 @@ def page_server_error(error):
 
 
 def convert_to_readable_time(time):
-    return time.strftime('%a %b %d,%Y %I:%M:%S %p')
-
-
-def display(forecast):
-    return 'Time: {}\nTemperature: {}°C\nHumidity: {}\nProbability of Precipitation: {}\nWeather Summary: {}'.format(convert_to_readable_time(forecast.time), round(forecast.temperature, 2), forecast.humidity, forecast.precipProbability, forecast.summary)
+    return time.strftime('%b/%d/%Y %a %I:%M:%S %p')
+
+
+def display(forecast, time_method):
+    if time_method == 'currently':
+        return 'Time: {}\nTemperature: {}°C\nHumidity: {}\nProbability of Precipitation: {}\nWeather Summary: {}\
+        '.format(convert_to_readable_time(forecast.time), round(forecast.temperature, 2), forecast.humidity, forecast.precipProbability, forecast.summary)
+
+    info = ''
+    count = 0
+    for item in forecast.data:
+        info += '\n  time: {}'.format(convert_to_readable_time(item.time))
+        info += '\n  temperature: {}'.format(item.temperature)
+        info += '\n  humidity: {}'.format(item.humidity)
+        info += '\n  precipProbability: {}'.format(item.precipProbability)
+        info += '\n  summary: {}\n'.format(item.summary)
```

Comment:
@dashmug they prefer that over += for string concatenation (due to the loop), which is fair enough. For that problem, you might use join(list) or cStringIO for all I care. I was trying to think about how to assemble this single fixed-size string. And there triple-quotes are viable, and Google does not disagree, see the bottom, though they prefer *implicit join*s. Which is another solution that i'd be fine with (it just invites ppl adding a comma :) ).

@admiral96 try implicit line joining, see how it looks.